canonical/assets.ubuntu.com
# feat: Implement new design for asset cards

## Done

- Makes a two new cards, one for images and one for everything else, based on [the figma](https://www.figma.com/design/dOz7GIHMmofSdje0l0pVFs/Asset-manager?node-id=0-1&node-type=canvas&t=16635YMhVjYQev9F-0)
- Fixes how we handle base 64 encoding, allowing reading of width and height on the back end
- Create a new route `/details` for a detailed view of each asset

## QA

- Run this branch locally
- Use the search to display some cards and see that the fields for image and others matches that of the design
- Text the 'full asset details' link works and goes to separate page where all the stored details can be seen
- Upload a new asset and see the width and height is calculated

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15205
